src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -1200,6 +1200,94 @@ public async Task TestSqlMetadataForInvalidConfigEntities()
             Assert.AreEqual("No stored procedure definition found for the given database object publishers", exceptionsList[1].Message);
         }
 
+        /// <summary>
+        /// This Test validates that when the entities in the runtime config have source object as null,
+        /// the validation exception handler collects the message and exits gracefully.
+        /// </summary>
+        [TestMethod("Validate Exception handling for Entities with Source object as null."), TestCategory(TestCategory.MSSQL)]
+        public async Task TestSqlMetadataValidationForEntitiesWithInvalidSource()
```

Comment:
Should we add a valid entity and then confirm that we don't have error messaging for the entity that should not be throwing any errors?

Reply:
there are already tests from previous PRs that verify valid entities with no exception messages. for example, `TestSqlMetadataForValidConfigEntities` in ConfigurationTests.cs